tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -6623,6 +6625,14 @@ TEST_P(OpConverter_FP32_Test, ConvertUnary) {
       {1, 1, 2, 3},  // expected output dims
   };
   for (const string& op_name : ops_to_test) {
+    // kSIGN  and kROUND are not yet supported in implicit batch mode.
+    // TODO(drivanov): remove following check when corresponding Myelin support
+    //                 will be implemented
+    if (trt_mode_ == TrtTestMode::kImplicitBatch &&
+        (op_name == "Sign" || op_name == "Round")) {
+      continue;  // skip testing
+    }
```

Comment:
We need to actually run the test, to ensure that the converter throws error in this combination, I recommend to introduce a conversion status variable.
```suggestion
    Status conv_status;
    if (trt_mode_ == TrtTestMode::kImplicitBatch &&
        (op_name == "Sign" || op_name == "Round")) {
         conv_status = errors::Unimplemented("Unary op: ", op_name,
                                 " not supported in implicit batch mode");
    }
```
Below (line 6656) you need to pass this to the test routine
```
  TestOpConverter("my_unary", node_def, p.expected_output_dims, conv_status,
                    p.runtime_status, ArrayFloatNear(output, 0.0001, true));
```

Reply:
Done